draft-ietf-quic-transport.md

```diff
@@ -1469,9 +1469,17 @@ of three ways:
    should send a Public Reset packet in return.  (TODO: articulate rules around
    when a public reset should be sent.)
 
-TODO: Connections that are terminated are added to a TIME_WAIT list at the
-server, so as to absorb any straggler packets in the network.  Discuss TIME_WAIT
-list.
+After receiving either a CONNECTION_CLOSE frame or a Public Reset, an
+endpoint MUST NOT send additional packets on that connection. After
+sending either a CONNECTION_CLOSE frame or a Public Reset packet,
+implementations MUST NOT send any non-closing packets on that connection. If
+additional packets are received after this time, implementations
+SHOULD respond to them by sending either a CONNECTION_CLOSE frame or a
+Public Reset packet, either of which may just be a duplicate of a
```

Comment:
I recommend that you use CONNECTION_CLOSE here and instead note that an endpoint MAY send a Public Reset instead if it doesn't want to save the state necessary to generate CONNECTION_CLOSE.

There's a sliding scale here:

1. maintain full state and generate a new CONNECTION_CLOSE in a new packet
2. only keep the packet that holds CONNECTION_CLOSE and retransmit that unmodified
3. drop state and rely on Public/Stateless Reset